common/src/main/resources/assets/plasmo_voice/lang/nl_nl.json

```
@@ -0,0 +1,191 @@
+{
+    "key.plasmo_voice.ptt": "Push-to-talk knop",
+    "key.plasmo_voice.priority_ptt": "Prioriteit push-to-talk",
+    "key.plasmo_voice.settings": "Spraakinstellingen",
+    "key.plasmo_voice.mute": "Microfoon dempen",
+    "key.plasmo_voice.mute_chat": "Schakel spraakchat uit",
+    "key.plasmo_voice.action": "Actieknop",
+
+    "key.plasmo_voice.distance": "Spreekafstand",
+    "key.plasmo_voice.distance.increase": "Vergroot spreekafstand",
+    "key.plasmo_voice.distance.decrease": "Verklein spreekafstand",
+
+    "key.plasmo_voice.occlusion": "Geluid occlusie",
+    "key.plasmo_voice.occlusion.toggle": "Geluidsocclusie in-/uitschakelen",
+
+    "gui.plasmo_voice.player_volume": "Speler volume: %s",
+    "gui.plasmo_voice.close": "Sluiten",
+    "gui.plasmo_voice.on": "Aan",
+    "gui.plasmo_voice.off": "Uit",
+
+    "gui.plasmo_voice.title": "Instellingen",
```

```suggestion
    "gui.plasmo_voice.title": "%s Instellingen",
```

Maybe this translation is not reflected. `%s` is required. Please see here for details: #108

(My suggestion doesn't understand Dutch very well, so I'm sorry if it's different.)

---

Good catch